src/CompletionProvider.ts

```diff
@@ -11,20 +11,136 @@ import {
 } from './utils';
 import type {CamelCaseValues} from './utils';
 
-// check if current character or last character is .
-function isTrigger(line: string, position: Position): boolean {
+export const COMPLETION_TRIGGERS = ['.', '[', '"', "'"] as const;
```

Comment:
Do we want to handle backtick `\`` as well?

Reply:
Adding both literal quotes (i.e. `'` and `"`) was a way to trigger the completion again as the editor would dismiss the completion options if you wrote the start of the string literal. I think we could add the backtick as well, as long as a literal class name is written (but not an interpolation expression) should still be handled, but would not provide extra value.

If it's ok with you, I would explore this as part of a following PR.